doc/antora/modules/howto/pages/troubleshoot.adoc

```diff
@@ -0,0 +1,77 @@
+#Troubleshooting
+
+Which operating system should I use?
```

Comment:
The questions / headlines here should be bold, otherwise they don't stand out from the answers.